search/search_repository_blackbox_test.go

```diff
@@ -3,14 +3,19 @@ package search_test
 import (
 	"context"
 	"fmt"
+	"math"
+	"strconv"
+	"strings"
 	"testing"
 
 	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/rendering"
 	"github.com/fabric8-services/fabric8-wit/resource"
 	"github.com/fabric8-services/fabric8-wit/search"
 	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
 	"github.com/fabric8-services/fabric8-wit/workitem"
 	"github.com/fabric8-services/fabric8-wit/workitem/link"
+	_ "github.com/lib/pq"
```

Comment:
Is thins blank import required here?

Reply:
@baijum good catch. I've removed it here 07e101d